integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/management/CarbonLoadDataCommand.scala

```diff
@@ -188,11 +188,13 @@ case class CarbonLoadDataCommand(
     val carbonLoadModel = new CarbonLoadModel()
     val tableProperties = table.getTableInfo.getFactTable.getTableProperties
     val optionsFinal = LoadOption.fillOptionWithDefaultValue(options.asJava)
+    // These two delimiters are non configurable and hardcoded for map type
+    optionsFinal.put("complex_delimiter_level_3", "\003")
+    optionsFinal.put("complex_delimiter_level_4", "\004")
```

Comment:
Remove hardcoded values. Create one `ComplexDelimeterEnum` and use the values from there

Reply:
done